plat/common/x86/cpu_native.c

```diff
@@ -64,11 +86,11 @@ void system_off(void)
 	 */
 	outw(0x604, 0x2000);
 
+#ifdef CONFIG_KVM_VMM_QEMU
 	/*
 	 * If that didn't work for whatever reason, try poking the QEMU
-	 * "isa-debug-exit" device to "shutdown". Should be harmless if it is
-	 * not present. This is used to enable automated tests on virtio.  Note
-	 * that the actual QEMU exit() status will be 83 ('S', 41 << 1 | 1).
+	 * "isa-debug-exit" device to "shutdown".
 	 */
-	outw(0x501, 41);
+	qemu_debug_exit(41);
```

Comment:
Would it make sense to have a macro for `41` - this doesn't tell me anything? Why `41`? What magic is this?